python/pyspark/testing/utils.py

```diff
@@ -49,6 +60,20 @@
 SPARK_HOME = _find_spark_home()
 
 
+class bcolors:
+    NC = "\033[0m"  # No Color, reset all
```

Comment:
`no_color`, `light_red` and `light_blue` per PEP 8, and to be consistent.
Optionally you can put `NO_COLOR`, `LIGHT_RED` and `LIGHT_BLUE` at the module level as a legitimate constant (that complies PEP 8).